examples/pipeline_wavernn/functional.py

```diff
@@ -0,0 +1,30 @@
+import torch
+
+
+def mulaw_encode(waveform, mu):
```

Comment:
Is [this](https://github.com/pytorch/audio/blob/master/torchaudio/transforms.py#L510) the same?

Reply:
I test them in notebook and they are same. I replaced it.